app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt

```diff
@@ -442,20 +452,35 @@ class PatientEntryScreen(context: Context, attrs: AttributeSet) : RelativeLayout
     }
   }
 
-    override fun showInvalidAgeError(show: Boolean) {
-      ageEditTextInputLayout.error = when {
-        show -> resources.getString(R.string.patiententry_invalid_age_error)
-        else -> null
-      }
+  override fun showAgeExceedsMaxLimitError(show: Boolean) {
+    ageEditTextInputLayout.error = when {
+      show -> resources.getString(R.string.patiententry_invalid_age_error)
+      else -> null
     }
+  }
 
-  override fun showInvalidDateOfBirthAgeError(show: Boolean) {
+  override fun showDOBExceedsMaxLimitError(show: Boolean) {
     dateOfBirthInputLayout.error = when {
       show -> resources.getString(R.string.patiententry_invalid_age_error)
```

Comment:
Rename the string resource to be specific.

Reply:
Done